torch/nn/parameter.pyi

```diff
@@ -1,4 +1,7 @@
 from .. import Tensor
+import builtins
 
 class Parameter(Tensor):
+    def __init__(self, data: Tensor, requires_grad: builtins.bool): ...
```

Comment:
Why do we need to use `builtin.bools`? Does `bool` work?

Reply:
On this I took inspiration on `torch/__init__.pyi.in`, without knowing the ins and outs of the decision that lead them to use this one rather than the simple `bool`.
Can revert to simple if need be.

Reply:
In `torch/__init__.pyi` we cannot use bool directly because we define `torch.bool`, and that means that the local definition always gets picked up over the builtin. I don't know if that applies here, but it's harmless enough to use builtins.